drake/doc/ubuntu.rst

```diff
@@ -30,13 +30,18 @@ Install the prerequisites::
     # Ubuntu 14.04 LTS (Trusty)
     sudo apt-get install cmake-curses-gui
 
+    # These are not required for the basic "make" rule.  We suggest that you
+    # wait to do this step until you actually need these tools.  (We intend
+    # to switch to non-pip instructions once that approach has been tested.)
     sudo pip install -U cpplint Sphinx
 
 Download the external dependencies::
 
     cd drake-distro
-    make options  # use the GUI to choose which externals you want, then press 'c' to configure, then 'g' to generate makefiles and exit
-    make download-all
+    env CXX=g++-4.9 CC=gcc-4.9 PATH=/usr/lib/ccache:$PATH make options
+    # Use the GUI to choose which externals you want, then press 'c' twice to configure, then 'g' to generate makefiles and exit.
+    # If you intend to modify Drake, we suggest that you set CMAKE_BUILD_TYPE to Debug instead of Release after the first 'g'.
```

Comment:
This should go on the developers page rather than lengthening the installation instructions. It applies to all platforms and we should aim to keep platforms in sync.